dist/s_style

```diff
@@ -59,11 +59,11 @@ else
 
 	# Finding paired typos in the comments of different file types and excluding invalid edge cases.
 	if [ "${fname##*.}" = "py" ]; then
-		egrep '#.*\b([a-zA-Z]+)\s\1\b' $f | egrep -v -e "in-memory" -e "one-file" > $t
+		egrep '#.*\b\s([a-zA-Z]+)\ \1\b[[:space:]\.]' $f > $t
 	elif [ "${fname##*.}" = "c" ] || [ "${fname##*.}" = "h" ]; then
-		egrep '/?*.*\b([a-zA-Z]+)\s\1\b' $f | egrep -v -e "@" -e "long long"> $t
+		egrep '/?\*.* \b([a-zA-Z]+) \1\b[[:space:]\.]' $f | egrep -v -e "@" -e "long long" > $t
```

Comment:
I would use \s to denote the space here in all three regex cases 😄

Reply:
Thanks, updated that! :)